src/viem/getLockableWallet.ts

```diff
@@ -69,7 +69,7 @@ export default function getLockableViemWallet(
     chain,
     transport: getTransport(chain),
     account,
-  }).extend((client: Client): Actions => {
+  }).extend((client: Client<Transport, Chain, Account>): Actions => {
```

Comment:
with our setup we always have a Transport, Chain, and Account. This added specificity makes TS play nice when calling `signMessage` b/c it now assumes (correctly) that there is a valid account attached to the wallet